content/attacks/posts/2022-01-17-Crypto.com.md

```diff
@@ -0,0 +1,32 @@
+---
+date: 2022-01-17
+custodians: Crypto.com
+categories: Custodian
+title: Crypto.com Confirms $34M Hack
+---
+
+## Summary
+
+According to the announcement, an attacker found a way to approve transactions without the two-factor authentication control being inputted by account holders. This resulted in 483 Crypto.com users losing funds from their accounts. The exchange reaffirmed comments made by the firm’s CEO, Kris Marszalek, that any accounts found to be impacted were fully restored, resulting in no loss of funds for users.
+
+## Attackers
+
+The identity of the attackers remained unknown.
+
+## Losses
+
+Attacker drained 4,836.26 ETH, 443.93 BTC, and approximately $66,200 of other currencies from its users’ accounts. The stolen funds total approximately $34.4 million.
+
+## Timeline
+
+On Monday, [Jan. 17](https://twitter.com/cryptocom/status/1482936866001207296), at 04:44 UTC, the company's twitter account announced unauthorized activity on a small number of user accounts. All withdrawals were paused for investigation. Infrastructure was up and running again in under 14 hours. Crypto.com CEO Kris Marszalek [tweeted](https://twitter.com/kris/status/1483277350683185155) confirmation that no customer funds had been lost. All users [asked](https://twitter.com/cryptocom/status/1483050866894868484) to sign back into their accounts and reset 2FA
+
+## Security Failure Causes
```

Comment:
use bullet point structure found in other published articles

Reply:
Done